app/uk/gov/hmrc/vatapi/resources/wrappers/ObligationsResponse.scala

```diff
@@ -59,4 +64,13 @@ case class ObligationsResponse(underlying: HttpResponse) extends Response {
         error("Non json response from DES")
     }
   }
+
+  override def errorMappings: PartialFunction[Int, Result] = {
+    case 400 if errorCodeIsOneOf(INVALID_IDTYPE, INVALID_IDNUMBER, INVALID_STATUS, INVALID_REGIME, NOT_FOUND_BPKEY) =>
```

Comment:
`INVALID_STATUS` would not be a 500 as this is supplied from the vendors software and they have the ability to change this so it would be a 400. Referring the GET Obligations RAML below: 

`Invalid status | 400 (Bad Request) | INVALID_STATUS`